ext/opentelemetry-ext-asyncpg/src/opentelemetry/ext/asyncpg/__init__.py

```diff
@@ -58,7 +61,16 @@ def _exception_to_canonical_code(exc: Exception) -> StatusCanonicalCode:
     return StatusCanonicalCode.UNKNOWN
 
 
-def _hydrate_span_from_args(connection, query, parameters) -> dict:
+def _mask_parameters(parameters):
```

Comment:
is this can impact on performance?

Reply:
Yeah, the both the serialization & hashing will have some overhead. I don't know a great way to define how much for this code path :S